<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization Interface</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/icons/search-icon-2.png</normaloff>:/Resources/icons/search-icon-2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Glyph View</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGraphicsView" name="view"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>List View</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="treemapSetting"/>
   <addaction name="separator"/>
   <addaction name="openVisualMapping"/>
   <addaction name="separator"/>
   <addaction name="crawlerSetting"/>
   <addaction name="separator"/>
   <addaction name="actionDataManager"/>
   <addaction name="actionCacheManager"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>300</width>
       <height>338</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>325</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>282</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Search</string>
      </attribute>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>243</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Resources/light-icons/appbar.magnify.browse.png</normaloff>:/Resources/light-icons/appbar.magnify.browse.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>282</width>
         <height>284</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name       </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="key"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Type         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="type"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Size          </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sizeFrom">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sizeFromUnit">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GB</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sizeTo">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sizeToUnit">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GB</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Modified</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fromMdt"/>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toMdt"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Created  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fromCtime"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toCtime"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Accessed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fromAtime"/>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toAtime"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Owner     </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="user"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Group      </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="group"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <zorder>layoutWidget</zorder>
      <zorder>searchButton</zorder>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>282</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Tools</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>178</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QPushButton" name="trainButton">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Train</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="openVisualMapping">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/light-icons/appbar.os.ubuntu.png</normaloff>:/Resources/light-icons/appbar.os.ubuntu.png</iconset>
   </property>
   <property name="text">
    <string>Visual Mapping</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="treemapSetting">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/light-icons/appbar.tiles.nine.png</normaloff>:/Resources/light-icons/appbar.tiles.nine.png</iconset>
   </property>
   <property name="text">
    <string>Treemap Setting</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="crawlerSetting">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/light-icons/appbar.copyright.png</normaloff>:/Resources/light-icons/appbar.copyright.png</iconset>
   </property>
   <property name="text">
    <string>Crawler</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionDataManager">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/light-icons/appbar.box.png</normaloff>:/Resources/light-icons/appbar.box.png</iconset>
   </property>
   <property name="text">
    <string>Simulated Data Manager</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionCacheManager">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/light-icons/appbar.box.layered.png</normaloff>:/Resources/light-icons/appbar.box.layered.png</iconset>
   </property>
   <property name="text">
    <string>Cache Manager</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
